### PR TITLE
Wiki: Increase font size and free space to make it more readable

### PR DESCRIPTION
**Pull request by mfrister**
This bugged me for a while. It's not perfect now, but I like it much better. Feedback welcome :)

op-wiki-space

The one with the larger font and more free space is the new version.

See: https://www.openproject.org/work_packages/8605

### CODE REVIEW

**Comment by tessi** [comment]
:thumbsup: for the gif :)

Apart from that I'd rather increase the overall font size instead of the wiki font size.

I like the change of the separator line width :thumbsup: 

Maybe we also want to introduce a left-padding for bullet points, a bigger space between paragraphs and no &lt;br&gt;'s for simple newlines (as in github markdown)

**Comment by myabc** [comment]
I agree this looks better, but…

> Apart from that I'd rather increase the overall font size instead of the wiki font size.

+1 for @tessi's comment. There are other places we should increase font size too.

File: app/assets/stylesheets/content/_wiki.sass [hunk 1]
```diff
@@ -28,6 +28,10 @@
 @import global/all
 
 div.wiki
+  font-size: 14px
```
Comment (myabc) [inline app/assets/stylesheets/content/_wiki.sass:31]: We should consider moving to to relative units throughout (be it percentage, `em` or `rem`), especially given our accessibility requirements/responsiveness goals.

@zzmedia thoughts?

**Comment by mfrister** [comment]
Added some further improvements for lists and headings. Updated the GIF above.

**Comment by tessi** [comment]
the related work package is https://www.openproject.org/work_packages/8605

**Comment by mfrister** [comment]
@myabc +1 for increasing the font size overall. I wasn't sure where I should do this and what I'd break, so I only changed the wiki.

If you all reassure me just changing that would be a good idea, I'll go ahead :)

**Comment by mfrister** [comment]
@zzmedia Any thoughts on this?

**Comment by zzmedia** [comment]
@myabc you're right, I think we should use `rem` in the future

**Comment by myabc** [comment]
@meeee could postpone merging this in/further discussion until next week (when @zzmedia is in Berlin)?

**Comment by mfrister** [comment]
@myabc Sure

**Comment by linki** [comment]
@myabc @meeee @zzmedia @lindenthal today, we decided not to wait for @zzmedia to merge this

**Comment by tessi** [comment]
:tada:

**Pull request merged**